integration-tests/src/test/resources/domainonpvwlst.yaml

```diff
@@ -16,5 +16,6 @@ exposeAdminNodePort: true
 namespace: default
 loadBalancer: TRAEFIK
 ingressPerDomain: true
+javaOptions: -Dweblogic.debug.DebugSituationalConfig=true -Dweblogic.debug.DebugSituationalConfigDumpXml=true
```

Comment:
This domain file is used in multiple tests, if you don't want the sit config debug to be enabled for all tests, you can add javaOptions in the domain map in IT class.

Reply:
Moved the javaOptions to createSitConfigDomain() method